src/Reader/StandaloneExtensionManager.php

```diff
@@ -9,22 +9,24 @@
 
 namespace Zend\Feed\Reader;
 
+use Zend\ServiceManager\Exception\InvalidServiceException;
```

Comment:
You cannot use ServiceManager exceptions in here. Use `Zend\Feed\Reader\Exception\InvalidArgumentException`

Reply:
But that's the type that is thrown?

Reply:
throw that other type

Reply:
OK, I see why.